vendor/go.etcd.io/bbolt/db.go

```diff
@@ -70,6 +80,13 @@ type DB struct {
 	// re-sync during recovery.
 	NoFreelistSync bool
 
+	// FreelistType sets the backend freelist type. There are two options. Array which is simple but endures
+	// dramatic performance degradation if database is large and framentation in freelist is common.
+	// The alternative one is using hashmap, it is faster in almost all circumstances
+	// but it doesn't guarantee that it offers the smallest page id available. In normal case it is safe.
+	// The default type is array
+	FreelistType FreelistType
```

Comment:
Looks like the default didn't change; no idea if changing is possible and/or if it would bring benefits in containerd?

Reply:
The default doesn't change yet, the new freelist type(hashmap) is experimental for now.

Reply:
The new freelist type will bring performance boost in almost all cases.